docs/deploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -e # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+REPO=$(git config remote.origin.url)
+SHA=$(git rev-parse --verify HEAD)
+GH_USER=dev10 # this needs to be replaced with a valid user id
```

Comment:
Remove this line, and add a comment in the doc that we need this, just like we need `GITHUB_TOKEN`. Travis has two formats for providing environment variables, one file based the other through their web—and possibly CLI—interface.